ambient-light/AmbientLightSensor_start.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>AmbientLightSensor Test: start()</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://www.w3.org/TR/ambient-light/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+var sensor, start_return;
+
+setup(function() {
+  sensor = new AmbientLightSensor();
+  start_return = sensor.start();
+});
+
+test(function() {
+  assert_equals(sensor.state, "activating");
```

Comment:
I think it is better to put all these assertions in this test file together within `ambient-light/AmbientLightSensor_onstatechange.html`.